spec/api_client_spec.rb

```diff
@@ -197,8 +197,8 @@
     let(:api_client) { Patch::ApiClient.new }
 
     it 'works' do
-      expect(api_client.select_header_content_type(nil)).to eq('application/json')
-      expect(api_client.select_header_content_type([])).to eq('application/json')
+      expect(api_client.select_header_content_type(nil)).to be_nil
+      expect(api_client.select_header_content_type([])).to be_nil
```

Comment:
this is expected?

Reply:
Yes. The generator changed this signature slightly. But it doesn't affect our calls since the content-type is specified elsewhere.